CriteoPublisherSdk/Sources/Standalone/CRInterstitial.m

```diff
@@ -184,6 +188,10 @@ - (void)loadAdWithBid:(CRBid *)bid {
     return;
   }
 
+  if (self.adUnit.adUnitId == nil && [bid.adUnit isKindOfClass:[CRInterstitialAdUnit class]]) {
```

Comment:
exactly same comment than for banner view. And as you're using such logic twice, I strongly suggest to add an internal consumeFor method in CRBid

Reply:
Added the `consumeFor` method.